kubetest/client.py

```diff
@@ -253,6 +253,28 @@ def load_persistentvolumeclaim(self, path, set_namespace=True):
             persistentvolumeclaim.namespace = self.namespace
         return persistentvolumeclaim
 
+    def load_ingress(self, path, set_namespace=True) -> objects.ingress:
+        """Load a manifest YAML into a Ingress object.
+
+        By default, this will augment the Ingress object with
+        the generated test case namespace. This behavior can be
+        disabled with the ``set_namespace`` flag.
+
+        Args:
+            path (str): The path to the Ingress manifest.
+            set_namespace (bool): Enable/disable the automatic
+                augmentation of the Ingress namespace.
+
+        Returns:
+            objects.Ingress: The ingress for the specified
+            manifest.
+        """
+        log.info('loading ingress from path: %s', path)
+        ingress = objects.ingress.load(path)
```

Comment:
I believe `ingress` here should be capitalized, since its referencing the `Ingress` object